frontend/src/metabase/visualizations/visualizations/EmbedViz/utils.ts

```diff
@@ -0,0 +1,104 @@
+import { isSafeUrl } from "metabase/lib/formatting/link";
+
+const embedLinkTransformers: {
+  test: (url: string) => boolean;
+  transform: (url: string) => string;
+}[] = [
+  {
+    test: url => url.includes("loom.com/share"),
+    transform: url => url.replace("/share/", "/embed/"),
+  },
+  {
+    test: url => url.includes("youtube.com/watch") || url.includes("youtu.be/"),
```

Comment:
What about youtube playlists?

We should find a library for this instead, it would probably also handle more services.

Reply:
Unfortunately, there is no such library

Reply:
https://github.com/metabase/metabase/pull/47994#issuecomment-2384860716